### PR TITLE
Add Automatic updater docs and screenshots to  manual

### PR DESCRIPTION
**Pull request by xh3n1**

### CODE REVIEW

File: ch034_upgrade_using_automatic_updater.xhtml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+"http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml"><head><title/></head><body>
+
+  <h1 id="automaticupdaterforphplist">Automatic updater for phpList</h1>
+
+  <p>Provides an easy, automated, web-based update mechanism for phpList installations.</p>
+
+  <h2 id="usage">Usage</h2>
+
+  <p>The new phpList updater gives you an easy way to upgrade your installation via web interface. In just four steps you can update your installation to the latest release. </p>
```
Comment (samtuke) [inline ch034_upgrade_using_automatic_updater.xhtml]: 'new' isn't necessary -- people reading the manual won't know if it's new, and it won't be new for long

File: toc.ncx [hunk 2]
```diff
@@ -189,6 +189,12 @@
         </navLabel>
         <content src="ch028_installation.xhtml"/>
       </navPoint>
+      <navPoint id="chapter34" playOrder="25">
+        <navLabel>
+          <text>Upgrading installation using Automatic Updater</text>
```
Comment (samtuke) [inline toc.ncx]: I would change to 'Upgrading using the Automatic Updater'

**Comment by samtuke** [comment]
Once this is merged, will you remove this text from the repository? Duplicate docs are not a good idea, but the manual can be the home for this long term, and only more technical info kept in the repo Readme.

File: ch034_upgrade_using_automatic_updater.xhtml (hunk 1 above)
Comment (xh3n1) [reply ch034_upgrade_using_automatic_updater.xhtml]: I agree.

**Comment by xh3n1** [comment]
@samtuke yes, I will update the readme after this gets merged.

**Pull request merged**